share/smack/top.py

```diff
@@ -110,6 +110,13 @@ def arguments():
   noise_group.add_argument('--debug-only', metavar='MODULES', default=None,
     type=str, help='limit debugging output to given MODULES')
 
+  noise_group.add_argument('--warn', default="unsound",
+    choices=['silent', 'unsound', 'info'],
+    help='''enable certain type of warning messages
+            (silent: no warning messages;
+            unsound: warnings about unsoundness;
+            info: warnings about unsoundness and debug information) [default: %(default)s]''')
```

Comment:
Same comment as above about `debug information`. We already have a flag `--debug` that turns on debug information.